### PR TITLE
feature(ui-ux): display latest blocks and transactions on landing page

### PR DESCRIPTION
**Pull request by lykalabrada**
#### Which issue(s) does this PR fixes?:

Fixes #

#### Developer Checklist:

- [x] Read your code changes at least once
- [x] Your UI implementation visually matched the rendered design\*
- [ ] Unit tests\*
- [ ] Added e2e tests\*

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -35,6 +35,7 @@
     "@types/react": "18.0.25",
     "@types/react-dom": "18.0.8",
     "@types/recharts": "^1.8.24",
+    "@typescript-eslint/eslint-plugin": "^5.42.1",
```
Comment (kyleleow) [inline package.json]: question: what is this for?

File: src/components/LatestDataTable.tsx [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+              </div>
+            </div>
+          </div>
+        </div>
+      </GradientCardContainer>
+    </div>
+  );
+}
+
+const truncateTextFromMiddle = (text: string, length = 5): string => {
```
Comment (kyleleow) [inline src/components/LatestDataTable.tsx]: Let's refactor this its own file

File: src/components/LatestDataTable.tsx [hunk 3]
```diff
@@ -0,0 +1,203 @@
...
+  const truncatedText = `${text.substring(0, length)}...${text.substring(
+    text.length - length,
+    text.length
+  )}`;
+  return truncatedText;
+};
+
+const DmxTokenSymbol = "DMXTc";
```
Comment (kyleleow) [inline src/components/LatestDataTable.tsx]: let's refactor this, being used in `TokenStatsDisplay` too

File: package.json (hunk 1 above)
Comment (lykalabrada) [reply package.json]: I was getting an error about this missing package before, but I tried just now and it's working without this. Removing now

Comment (lykalabrada) [reply package.json]: Removed

File: src/components/LatestDataTable.tsx (hunk 3 above)
Comment (lykalabrada) [reply src/components/LatestDataTable.tsx]: Updated, moved to `constants.ts` file

File: src/components/LatestDataTable.tsx (hunk 2 above)
Comment (lykalabrada) [reply src/components/LatestDataTable.tsx]: Updated, moved to shared file

File: package.json (hunk 1 above)
Comment (kyleleow) [reply package.json]: Can we revert package-lock changes too?

File: src/components/LatestDataTable.tsx [hunk 4]
```diff
@@ -0,0 +1,196 @@
...
+  rowData: RowData;
+  amountPrefix: string;
+  rowIndex: number;
+}): JSX.Element {
+  const { transactionId, tokenAmount, datetime, txnOrBlockInfo } = rowData;
+  const detailsPageLink = `/${type}/${transactionId}`;
+  return (
+    <div
+      data-testid={`latest-${type}-row-${rowIndex}`}
+      className="text-white-50 py-5 border-b border-black-600/50 md:flex md:flex-wrap md:items-start lg:flex-nowrap xl:gap-5"
```
Comment (kyleleow) [inline src/components/LatestDataTable.tsx]:
```suggestion
      className="text-white-50 py-5 border-b border-black-600 md:flex md:flex-wrap md:items-start lg:flex-nowrap xl:gap-5"
```

**Review by kyleleow — APPROVED** [review]

**Pull request merged**